### DISCUSSION TITLE
Yakuza 0 and Kiwami PS3 version

### DISCUSSION THREAD

**Discussion by MasterMegatron** [Kaplas80/TF3#62] [Q&A]
Hello! Is it possible to use the framework to translate PS3 versions of games? Most of the game archives are the same, maybe some of the text is located in a different place. In any case, is there a way to add such functionality? And one more thing: is this framework only suitable for Latin translations? I want to translate these games into my language, but it is in Cyrillic.

**Comment by Kaplas80** [comment]
Hello!

Yakuza 0 or Kiwami can't be translated with TF3, only with TF2. And only the PC versions. I don't know if it would be easy to extract and repack the text for the PS3 version, but, in PC, it also need patching the .exe (at least for Spanish characters) and I don't know how to do in the PS3 .elf file.

I wrote a FAQ file with answers to some of your questions, but I misplaced it 😅
https://github.com/Kaplas80/TF3/blob/main/docs/faq.md

Regards,
